src/Sinch/Conversation/Messages/Send/SendMessageRequest.cs

```diff
@@ -154,11 +157,34 @@ public override string ToString()
             sb.Append("  ConversationMetadata: ").Append(ConversationMetadata).Append("\n");
             sb.Append("  Queue: ").Append(Queue).Append("\n");
             sb.Append("  Recipient: ").Append(Recipient).Append("\n");
-            sb.Append("  Ttl: ").Append(Ttl).Append("\n");
+            sb.Append("  Ttl: ").Append(TtlSeconds).Append("\n");
             sb.Append("  ProcessingStrategy: ").Append(ProcessingStrategy).Append("\n");
             sb.Append("  CorrelationId: ").Append(CorrelationId).Append("\n");
             sb.Append("}\n");
             return sb.ToString();
         }
     }
+
+    public class TimeToLiveConverter : JsonConverter<int?>
+    {
+        public override int? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            var str = reader.GetString();
+            if (str is null) return null;
+            str = str.Trim();
+            var val = str.TakeWhile(x => x == 's').ToString();
```

Comment:
And looking at the string conversion, I can't see an override of ToString() for IEnumerable so it would use the Object.ToString() et return the FQTN of your IEnumerable instance.
But I see a [ToArray() method](https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.toarray?view=net-8.0#system-linq-enumerable-toarray-1(system-collections-generic-ienumerable((-0)))) that would be useful before creating a new string from it.